siem.py

```diff
@@ -197,6 +203,8 @@ def main():
         endpoint_config['address'] = cfg.address.strip()
         endpoint_config['socktype'] = cfg.socktype.strip()
 
+    SIEM_LOGGER.addHandler(create_siem_log_handler(endpoint_config))
```

Comment:
The name `create_siem_log_handler` is a bit ambiguous.

Reply:
Updated method name in https://github.com/sophos/Sophos-Central-SIEM-Integration/pull/27/commits/d59c9991377ee8662af65e52fe751ef4e90ccea9